src/app/pages/component-sidenav/component-sidenav.ts

```diff
@@ -49,7 +49,7 @@ import {NavigationFocusService} from '../../shared/navigation-focus/navigation-f
 // These breakpoint values need to stay in sync with the related Sass variables in
 // src/styles/_constants.scss.
 const EXTRA_SMALL_WIDTH_BREAKPOINT = 720;
-const SMALL_WIDTH_BREAKPOINT = 959;
+const SMALL_WIDTH_BREAKPOINT = 939;
```

Comment:
I think that it's problematic that this small breakpoint was changed so that it no longer matches our Sass small breakpoint:

https://github.com/angular/material.angular.io/blob/53965ba3b8d271e4b7b29a8923f2c2ddba5dc488/src/styles/_constants.scss#L11

It also no longer matches any of the established Material Design layout breakpoints:
https://material.io/design/layout/responsive-layout-grid.html#breakpoints

I'm looking at reverting this in https://github.com/angular/material.angular.io/issues/257.